woocommerce/woocommerce-android
# [Tablet Orders] Add button and totals UI

Closes: #10425


### Description

The PR adds ViewModel of the data that we show on the bottom sheet and it's rendering in compose
Notice:
* Order Totals show on the order creation screen as not sure yet if we make the bottom sheet permanent or not 
* Loading is missing yet
* There is no actual mapping of the order data to the VM yet
* Buttons are not clickable yet

### Testing instructions

* Orders -> Create order
* Add product or custom amount
* Notice the bottom sheet with data

Make sure that there is not regression on the release build.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/38716be9-c9db-41d2-b585-f6b60e178fbc
